funnel/assets/js/schedule_view.js

```diff
@@ -23,7 +23,7 @@ const Schedule = {
         pageDetails: {
           url: window.location.href,
           title: $('title').html(),
-          projectTitle: $('title').html().split(' — ')[1],
+          projectTitle: $('title').html().split(' – ')[1],
```

Comment:
Don't do this. [Pass the unvarnished title separately in a title meta tag and read from that](https://stackoverflow.com/questions/21076201/difference-between-meta-name-title-tag-and-title-title-tag).

Reply:
Use `dc.title` for the meta tag. I think Baseframe does this already, as we've implemented the Dublin Core metadata set since the beginning.

Reply:
Fixed. Use `{{ project.title }}` from backend.